src/main/java/competition/subsystems/drive/commands/SwerveDriveWithJoysticksCommand.java

```diff
@@ -155,7 +155,11 @@ public void execute() {
         
         // Michael wants the triggers to be more precise pretty much all the time, so adding a trigger-only
         // power reduction
-        humanRotatePowerFromTriggers *= triggerOnlyPowerScaling.get();
+        if (drive.isUnlockFullDrivePowerActive()) {
+            // do nothing - unleash the full power of the machine!
+        } else {
+            humanRotatePowerFromTriggers *= triggerOnlyPowerScaling.get();
```

Comment:
You also need to account for 
`suggestedRotatePower *= turnPowerFactor.get();`
a few lines down.